test/file11.tsx

```diff
@@ -0,0 +1,129 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import {jsx} from '@emotion/react';
+import {Collapse} from '@xsite-ui/common.collapse';
```

Comment:
:robot: OpenAI

It's not clear what `Collapse` is or where it's coming from. Please add a comment or documentation explaining this.

<!-- This is an auto-generated comment by OpenAI -->